tests/test_repair.py

```diff
@@ -0,0 +1,30 @@
+import geopandas
+import maup
+
+# These tests are losely based off the test_example_case in test_prorate.py
+
+def test_example_close_gaps_repair_MI():
+    shp = geopandas.read_file("zip://./examples/MI.zip") # MI shapefile
+
+    holes = maup.repair.holes_of_union(shp)
+    assert len(holes) > 0
+    assert holes.unary_union.area > 100
+    shp["geometry"] = maup.close_gaps(shp, relative_threshold=None)
+
+    # assert len(maup.repair.holes_of_union(shp)) == 0 # this fails, probably due to floating precision issues
```

Comment:
What's the right behavior in this case, do you think? The contract for `close_gaps` says "Set `relative_threshold=None` to close all gaps"; it could be confusing if this is often violated. Two easy options:
* Relax the contract. ("Set `relative_threshold=None` to close all gaps, up to <some very small numerical tolerance>.")
* Raise an error when the gaps can't be closed with `relative_threshold=None`.

Reply:
Yep! I updated the [documentation](https://github.com/mggg/maup/pull/26/commits/fe353c4ef126f368af0a15391ce95a75abd94669#diff-53416a448fa906d7998c78f7ec0c817d22243317429fae7709d8f248d337e082) to relax the contract.